pkg/controller/certificaterequests/acme/acme.go

```diff
@@ -200,11 +200,22 @@ func (a *ACME) Sign(ctx context.Context, cr *v1.CertificateRequest, issuer v1.Ge
 
 // Build order. If we error here it is a terminating failure.
 func buildOrder(cr *v1.CertificateRequest, csr *x509.CertificateRequest) (*cmacme.Order, error) {
```

Comment:
Maybe add some unit tests for `buildOrder`.

Reply:
Added